eng/linkerTests/linkerCorrectnessTests.targets

```diff
@@ -0,0 +1,110 @@
+<Project>
+  <PropertyGroup>
+    <LinkerTestDir>$(ArtifactsBinDir)linkerTests/</LinkerTestDir>
+    <LinkerTestProjectsDir>$(LinkerTestDir)projects/</LinkerTestProjectsDir>
+    <TestDotNetPath>$(DotNetRoot)dotnet</TestDotNetPath>
+    <ProjectTemplate>$(MSBuildThisFileDirectory)project.csproj.template</ProjectTemplate>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TestConsoleAppSourceFiles Include="$(MSBuildProjectDirectory)/*.cs" />
+
+    <TestSupportFiles Include="$(MSBuildThisFileDirectory)SupportFiles/Directory.Build.props">
```

Comment:
Perhaps use a wildcard? `$(MSBuildThisFileDirectory)SupportFiles/Directory.Build.*`?

Reply:
yup, that would work too. I'll fix that.